src/pages/EnablePayments/AdditionalDetailsStep.js

```diff
@@ -232,7 +233,7 @@ class AdditionalDetailsStep extends React.Component {
                                         {this.props.translate('common.noPO')}
                                     </Text>
                                 </View>
-                                <TextInput
+                                <PhoneTextInput
```

Comment:
It seems that this component is lacking the validation for PhoneNumber. Could you please add that?

Reply:
I think we are already handling the validation in backend and showing the error (if any)
cc: @puneetlath @parasharrajat

Reply:
Hm, I would think we still want to do some front-end validation, even if we are also validating on the back-end.